CONTRIBUTING.md

```diff
@@ -486,36 +487,53 @@ if possible provide GPG signed commits.
 https://help.github.com/articles/securing-your-account-with-two-factor-authentication-2fa/
 https://help.github.com/articles/signing-commits-with-gpg/
 
+Furthermore, any account with write access (like bots and TOC members) **must** use 2FA.
```

Comment:
I would maybe even go as far as require 2FA for every maintainer. After all, it is not that hard with GitHub nowadays....

Reply:
There is currently a blocker to require this for accessibility reasons and it is presently being investigated.

Reply:
Oh I was completely sure that GitHub offers email 2FA which should be somehow doable for everyone, but seems like they don't, only SMS messages 😕
Then I have to agree with you